web/src/components/sidenav/Sidenav.tsx

```diff
@@ -74,13 +92,10 @@ class Sidenav extends React.Component<SidenavProps> {
             <RouterLink to={'/events'} className={classes.link}>
               <MqIconButton
                 id={'eventsButton'}
-                title={'EVENTS'}
+                title={i18next.t('sidenav.events')}
                 active={this.props.location.pathname === '/events'}
               >
-                <SVG
-                  src="https://raw.githubusercontent.com/MarquezProject/marquez/main/web/src/img/iconSearchArrow.svg"
-                  width={'30px'}
-                />
+                <SVG src='../../img/iconSearchArrow.svg' width={'30px'} />
```

Comment:
@merobi-hub, this reverts the change for https://github.com/MarquezProject/marquez/pull/2280